python/ray/_private/runtime_env/context.py

```diff
@@ -64,12 +64,16 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
             passthrough_args = class_path_args + passthrough_args
         elif language == Language.JULIA:
             executable = "julia"
+            julia_args = ["--project=/Users/cvogt/.julia/dev/ray_core_worker_julia_jll"]
```

Comment:
Just to get started but we'll need to figure out a better way

Reply:
I spent some time on this and we're now specifying the `JULIA_PROJECT` environmental variable to specify the project. This value is specified via the task specification from the Julia process which submitted the task. Overall this works great with the caveat that you cannot prestart workers unless the `ray_core_worker_julia_jll` package is installed globally